### DISCUSSION TITLE
cilium-handbook/networking/fragmentation/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/cloud-native-library#83] [Announcements]
# cilium-handbook/networking/fragmentation/

默认情况下，Cilium 将 eBPF 数据路径配置为执行 IP 分片跟踪，以允许不支持分片的协议（例如 UDP）通过网络透明地传输大型消息。IP 分片跟踪在 eBPF 中使用 LRU（最近最少使用）映射实现，需要 Linux 4.10 或更高版本。可以

https://lib.jimmysong.io/cilium-handbook/networking/fragmentation/

**Comment by OneDrawer** [comment]
`纤毛碎片跟踪需要第一个逻辑碎片首先到达。`

纤毛？

**Reply by rootsongjc** [reply]
已修改
